src/main/kotlin/step4/model/Race.kt

```diff
@@ -0,0 +1,33 @@
+package step4.model
+
+import step4.strategy.MoveStrategy
+
+data class Race(val names: String) {
+    private val cars = getCars(names)
+
+    fun makeTurn(strategy: MoveStrategy) {
+        for (car in cars) {
+            val move = strategy.resultOfTurn
+
+            car.save(move)
+        }
+    }
+
+    private fun getCars(names: String): MutableList<Car> {
+        val carNames = names.split(CAR_NAME_DELIMITER)
+        val carList = ArrayList<Car>()
+
+        for (carName in carNames) {
+            carList.add(Car(carName, 0, mutableListOf()))
+        }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val carList = carNames.map { Car(it, 0, mutableListOf()) }
```

Reply:
수정하였습니다. 코틀린에서 지원하는 부분을 더 활용할 수 있도록 해야겠네요